cEP-0008.md

```diff
@@ -0,0 +1,26 @@
+# Review workflow
+
+| Metadata     |                                             |
+| ------------ |---------------------------------------------|
+| cEP          | 8                                           |
+| Version      | 1.0                                         |
+| Title        | Review Workflow                             |
+| Authors      | Meet Mangukiya <meet123mangukiya@gmail.com> |
+| Status       | Proposed                                    |
+
+# Abstract
+
+Code Review at coala is done using GitHub pull requests. There is no feature to
+know the state of a review comment. This cEP aims at forming a standard review
+system so that new reviewers can get an idea of existing review comments and
+their states.
+
+# Proposed Change
+
+To solve this problem I propose, we use the comment reactions:
+1. :+1: for **acknowledged/addressed**.
+2. :-1: for **invalid/dropped review comments**
+3. :confused: for **under discussion**.
+
+Both reviewer and author has to react to the main/top-level comment on the given
+thread as a status report.
```

Comment:
imo we can add here that "A PR may not be merged until all the review comments have been acknowledge by both the reviewer and the author"